### ISSUE TITLE
[FEATURE] Allow to see search suggestion from browser when it's the primary search engine

### ISSUE THREAD

**Issue by fakeid30** [benbusby/whoogle-search#147]
**Describe the feature you'd like to see added**
Allow to see search suggestion from browser when it's the primary search engine like duckduckgo or google has. Not sure if it's possible. 

If not possible then it's fine. 

If possible but not time permitting, I would like to do it myself, any pointer on how to do it will be useful. 

Thank you.

**Comment by benbusby** [comment]
Is this what you're referring to?

![image](https://user-images.githubusercontent.com/33362396/99309320-3c927980-2827-11eb-9e88-34eab296be51.png)

If so, this is already supported. What browser are you using? Also what release of Whoogle (i.e. beta/develop vs latest/master)? The current develop branch of the project (or beta tag on Docker Hub) was recently patched with an opensearch fix that is likely relevant here if you're having issues seeing the search suggestions.

**Comment by fakeid30** [comment]
I'm using stable branch with Firefox browser and heroku deployment. I'll wait for an update.

**Comment by benbusby** [comment]
Yes, I just verified that the `heroku-app-beta` branch works, so you can switch to that one for now.

**Comment by FoxxMD** [comment]
Do I need to do anything to "active" opensearch on chrome? (using beta tag on dockerhub)

**Comment by benbusby** [comment]
@FoxxMD typically no. I did a quick test with Chrome after seeing your message and noticed the opensearch template was still getting rejected.  

I just pushed a519de90af39fb3825b4699cda70155f421ddfc8 which should finally resolve this issue on Chrome though. Once the beta tag on Docker Hub is updated, try with that and let me know if it works -- just be sure to remove any conflicting search engine entries in Chrome beforehand. It should automatically detect the search engine and have both search and search suggestions working properly. 

I manually tested the new code out with Chromium like 1000 times before pushing, so I'm fairly confident that it should work now.
